dotCMS/src/main/java/com/dotmarketing/business/ajax/PermissionAjax.java

```diff
@@ -350,7 +350,7 @@ public void resetAssetPermissions (String assetId, Long languageId) throws DotDa
 	}
 
 	@CloseDBIfOpened
-	private Permissionable retrievePermissionable (String assetId, Long language, User user, boolean respectFrontendRoles) throws DotDataException, DotSecurityException {
+	public static Permissionable retrievePermissionable (String assetId, Long language, User user, boolean respectFrontendRoles) throws DotDataException, DotSecurityException {
```

Comment:
I am not sure if you need this method on the actionlet since the only permissionable you can reset with workflow is a contentlet and it is already contained in the processor, that said, rollback this change and use the actual contentlet